lsp/tests/json_schema_errors.rs

```diff
@@ -0,0 +1,337 @@
+use awsdocuments_language_server::parsers::json_schema::{
```

Comment:
Should these tests live in `json_schema_parser.rs` instead of a separate directory/file? That tends to be the Rust way of doing things.